packages/paste-website/src/pages/components/card/index.mdx

```diff
@@ -0,0 +1,160 @@
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/card"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: { slug: { eq: "/card/" } }) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Card"
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/blob/master/packages/paste-core/components/card/"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/components-card--default"
+  data={props.data.allPasteComponent.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+## Guidelines
+
+### About Cards
+
+Cards are a specifically-styled container that group relatable content and actions. 
+Card is an extremely flexible container that does not require specific components inside of it, 
+allowing an implementer to compose as needed, but elements such as Title, Body, 
+and Button or Anchor are commonly used. See Examples for common Card patterns. 
+A Card does not currently handle any interactive events such as hover, click or focus, 
+though children composed inside of it may commonly have event handlers.
+
+Acknowledging the flexibility of Card component, there are several non-negotiable 
+properties of a Card that differentiate it from a more basic page-layout element, such as Box:
+
+* A background color of $background-color-body.  
+* A border width of $border-width-20.
+* A border radius of $border-radius-20.
+* A border color of $color-border.
+* A flexible width and height.
+    * Height is determined by a Card’s child components
+    * Width is determined by the parent component or container surrounding Card
+
+### Card vs. Box
+
+A card is a box with default attributes defined above. The purpose of the card is to provide a consistent experience to use across the website the design team can 
+
+
+## Usage Guide
+
+
+### Installation
+
+```bash
+yarn add @twilio-paste/card
+
+or
+
+npm i @twilio-paste/card
+```
+
+### API
+
+| Prop         	| Type             	| Description                              	| Default        	|
+|--------------	|------------------	|------------------------------------------	|----------------	|
+| children?    	| React.ReactNode  	| Child components to render into the card 	| undefined         |
+| borderWidth  	| borderWidths     	| Width of the border                      	| borderWidth20  	|
+| borderColor  	| BorderColors     	| Color of the border                      	| colorBorder    	|
+| borderStyle  	| BorderStyleProps 	| Style of the border                      	| solid          	|
+| borderRadius 	| Radii            	| Radius of the border                     	| borderRadius20 	|
+| padding      	| Spacing          	| Internal spacing of the card             	| space60        	|
```

Comment:
This prop table is out of date a little bit, you only allow "Global HTML attributes" and `PaddingProps` now